rust/phoenix-sdk/src/sdk_client.rs

```diff
@@ -1001,4 +1004,114 @@ impl SDKClient {
         );
         Ok(limit_order_ix)
     }
+
+    // Create a post-only order instruction from a PostOnlyOrderTemplate
+    pub async fn get_post_only_ix_from_template(
```

Comment:
Echoing a previous comment that this function probably should not be async

Reply:
Yep I agree. All 3 template -> instruction builders are now sync and accepts a MarketMetadata - commit [04bd048](https://github.com/Ellipsis-Labs/phoenix-sdk/pull/87/commits/04bd04814079e23024825eb140c913327833236c)